marketplace.xml

```diff
@@ -7168,4 +7168,44 @@ Pentaho currently does not provide a mechanism allowing for plugin steps to be d
      </versions>
    </market_entry>
    
+   <market_entry>
+    <id>CM4</id>
```

Comment:
This should match the plugin folder name and case must match. The folder name is cm4, so that should be the plugin id.